README.md

```diff
@@ -10,17 +10,31 @@ Test via ```nosetests```.
 ### Usage: 
     POST   on /shopcarts: creates new shopcart.
         Expects JSON body data:
-            item_id: -1
+            item_id: 0
             user_id: int
+            item_name: string, "ring" + str(item_id)
```

Comment:
This should just be any string. The 'ring' thing is just what our testing factory creates

Reply:
Fixed